### DISCUSSION TITLE
Why LND don't sync the graph in neutrino backend

### DISCUSSION THREAD

**Discussion by kannapoix** [lightningnetwork/lnd#8218] [Q&A] [answered]
I have a similar situation to this issue.
I understand that switching backends isn't supported, but I wonder why LND doesn't sync the graph in Neutrino mode.
[Switching from `neutrino` to `bitcoind`: is it stuck? · Issue #6900 · lightningnetwork/lnd](https://github.com/lightningnetwork/lnd/issues/6900)

AFAIK, the reason of long startup time when switching to bitcoind from neutrino as described in the issue is that LND starts syncing the graph from a very old block.

The questions are
1. How does LND maintain the channel graph in Neutrino mode without syncing?
2. Is there any way to speed up the sync or just catch up to the latest channel graph immediately?

**Comment by guggero** [comment, marked as answer]
`lnd` does sync the graph in Neutrino mode. But it doesn't verify each announced channel funding transaction on-chain, as that would mean downloading most blocks which would be very slow. So some data in the channel graph is missing if you use Neutrino.
Once you switch to a full chain backend, `lnd` will attempt to fill in that missing data which I think is what takes that long. While possible in theory, there is no way to speed this up with the currently implemented feature set.

**Reply by kannapoix** [reply]
>But it doesn't verify each announced channel funding transaction on-chain, as that would mean downloading most blocks which would be very slow.

I see, it makes sense.
Thank you very much.
